src/fu-util.c

```diff
@@ -2524,7 +2529,10 @@ main (int argc, char *argv[])
 		{ "no-history", '\0', 0, G_OPTION_ARG_NONE, &no_history,
 			/* TRANSLATORS: command line option */
 			_("Do not write to the history database"), NULL },
-		{ "show-all-devices", '\0', 0, G_OPTION_ARG_NONE, &priv->show_all_devices,
+		{ "show-all", '\0', 0, G_OPTION_ARG_NONE, &priv->show_all,
```

Comment:
make sure to update the bash completion for both `fwupdtool` and `fwupdmgr`

Reply:
And fish completion I guess uses this too

Reply:
Eeek, done.